examples/BERT/data.py

```diff
@@ -0,0 +1,79 @@
+import glob
+import torch
+import logging
+from torchtext.data.utils import get_tokenizer
+import random
+
+
+class LanguageModelingDataset(torch.utils.data.Dataset):
```

Comment:
Can't you use https://github.com/pytorch/text/blob/master/torchtext/experimental/datasets/language_modeling.py#L15 ?

Reply:
Switched to the one in `experimental/datasets`.